.changeset/twenty-cameras-take.md

```diff
@@ -0,0 +1,10 @@
+---
+"@fluid-experimental/presence": minor
+---
+---
+"section": other
+---
+
+Untangle `PresenceStates` methods and properties
+
+`PresenceStatesEntries` which represent each of the states in `PresenceStates` schema have been relocated from directly within `PresenceStates` to under property names `prop`. Only the `add` method remains. (Type `PresenceStatesMethods` has been removed.)
```

Comment:
````suggestion
The methods and properties of PresenceStates have been reorganized

The `PresenceStatesEntries` object, which represent each of the states in the `PresenceStates` schema, has been moved from directly within `PresenceStates` to under property names `props`. Only the `add` method remains directly within `PresenceStates`. The type `PresenceStatesMethods` has also been removed since it is no longer used.

To update existing code, access your presence states from the `props` property instead of directly on the `PresenceStates` object. For example:

```patch
- presenceStatesWorkspace.latestMapValueManager.local.get("key1");
+ presenceStatesWorkspace.props.latestMapValueManager.local.get("key1");
```
````

Reply:
Thanks - I will apply edits in a subsequent PR to allow merge and peeps to resolve conflicts.